public/employers/the-daily-show.yml

```diff
@@ -0,0 +1,29 @@
+name: The Daily Show
```

Comment:
For TV shows, are we going by the title of the show? TDS has two production companies involved along with other studios.

Reply:
With Ellen DeGeneres, I went with the name of her digital production firm, but not Warner Bros (since they weren't responsible for the decisions). In the case of TDS, I'm not sure what to go with since I don't know who handles the employees (hiring, payments, etc). Do you know which production company does it?

Reply:
No, though from the show's many segments and gags, it seems Viacom handles that sort of stuff. I'm not 100% sure on this though.

Reply:
Okay, I'll check into Viacom. Will re-request review if I have to update the employer.

Reply:
@frosteeze Updated based on production company Day Zero, which is owned by Noah, partnered with Viacom, and is responsible for the show's production.